content/guides/hmr-react.md

```diff
@@ -197,8 +247,34 @@ if (module.hot) {
 }
 ```
 
+The important thing to note in the code above is the `module` reference.
+
+1. webpack will expose `module.hot` to our code since we set `devServer: { hot: true }`;
+
+2. Thus we can use the `module.hot` hook to enable HMR for specific resources (Here's `App.js`). The most important API here is `module.hot.accept`, which specifies how to handle changes to specific dependencies.
+
+3. Note that because webpack 2 has built-in support for ES2015 modules, you won't need to re-require your root component in `module.hot.accept`. To make this work, you need to change the Babel ES2015 preset in `.babelrc` to be:
```

Comment:
It's better if you can just point to the Babel article and note that you should run it without modules enabled as webpack will take care of that. It's better to have Babel bits in a dedicated article as that goes beyond HMR.